### PR TITLE
SDK-1007: move enum to contracts

### PR DESCRIPTION
**Pull request by vol4onok**

### CODE REVIEW

File: auth.json [hunk 1]
```diff
@@ -1 +1,5 @@
-{}
+{
```
Comment (pavelmaksimov25) [inline auth.json:1]: Please remove

**Review by pavelmaksimov25 — CHANGES_REQUESTED** [review]

**Review by sergeyspryker — DISMISSED** [review]

File: src/Extension/Resources/config/task/AcpAppAsyncApiAddMessageSchemaTask.yaml [hunk 2]
```diff
@@ -5,15 +5,15 @@ help: ~
 stage: build
 version: 0.2.0
 command: "php %executable_command% schema:asyncapi:message:add %channel-name% %message-name% %operation-id% %asyncapi-file% %property% %from-transfer-class% %message-type% %add-metadata% -v"
-type: local_cli_interactive
+type: !php/const SprykerSdk\SdkContracts\Enum\Task::TYPE_LOCAL_CLI_INTERACTIVE
```
Comment (pavelmaksimov25) [inline src/Extension/Resources/config/task/AcpAppAsyncApiAddMessageSchemaTask.yaml:8]: Task and task set versions must be increased respectively. Required for all the tasks and task sets.

File: src/Extension/Task/GenerateAppTask.php [hunk 3]
```diff
@@ -65,7 +66,7 @@ public function getPlaceholders(): array
                 [
                     'name' => 'app-name',
                     'description' => 'Input name for new App',
-                    'type' => 'string',
+                    'type' => ValueTypeEnum::TYPE_STRING,
```
Comment (pavelmaksimov25) [inline src/Extension/Task/GenerateAppTask.php:69]: Task and task set versions must be increased respectively. The same for the rest of php commands

**Review by pavelmaksimov25 — CHANGES_REQUESTED** [review]

File: src/Extension/Resources/config/task/AcpAppAsyncApiAddMessageSchemaTask.yaml (hunk 2 above)
Comment (vol4onok) [reply src/Extension/Resources/config/task/AcpAppAsyncApiAddMessageSchemaTask.yaml:8]: I updated but for DB nothing were changed

File: composer.json [hunk 4]
```diff
@@ -32,7 +32,7 @@
         "spryker-sdk/acp": "^0.2.0",
         "spryker-sdk/async-api": "^0.2.3",
         "spryker-sdk/composer-replace": "dev-master",
-        "spryker-sdk/sdk-contracts": "^0.4.1",
+        "spryker-sdk/sdk-contracts": "dev-feature/sdk-1007-move-enum-to-contracts as 0.4.1",
```
Comment (pavelmaksimov25) [inline composer.json:35]: Let's wait for the sdk-contracts release and then update dependency version

**Review by sergeyspryker — APPROVED** [review]

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**